src/components/molecules/ChatPoll/ChatPoll.tsx

```diff
@@ -167,7 +150,7 @@ export const ChatPoll: React.FC<ChatPollProps> = ({
       <ChatMessageInfo
         message={message}
         reversed={isMine}
-        deleteMessage={deleteThisPollMessage}
+        deleteMessage={deletePollMessage ? deleteThisPollMessage : undefined}
```

Comment:
```suggestion
        deleteMessage={deletePollMessage && deleteThisPollMessage}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1963/commits/c6b97bed424271dbb74a0353cb2bb89dfb540c05